config/chime_science_run_gpu_no_output.yaml

```diff
@@ -459,6 +459,14 @@ gpu_rfi_output_buffers:
     kotekan_buffer: standard
   gpu_rfi_output_buffer_3:
     kotekan_buffer: standard
+  rfi_value_buffer_0:
+    kotekan_buffer: standard
+  rfi_value_buffer_1:
+    kotekan_buffer: standard
+  rfi_value_buffer_2:
+    kotekan_buffer: standard
+  rfi_value_buffer_3:
+    kotekan_buffer: standard
```

Comment:
This needs to be updated to match the production config.

Reply:
Ah good point.